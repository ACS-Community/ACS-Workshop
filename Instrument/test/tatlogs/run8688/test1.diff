0a1,7
> 1 - 
> 1 - ----------------------------------------------------------------------
> 1 - .
> 1 - 2014-09-09T19:06:43.515 Python Client: initialized by teami@instrument disconnect - Disconnect called for client
> 1 - CAMERA ON
> 1 - OK
> 1 - Ran 1 test in 0.176s

