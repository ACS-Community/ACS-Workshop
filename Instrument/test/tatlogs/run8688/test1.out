1 - .
1 - ----------------------------------------------------------------------
1 - Ran 1 test in 0.176s
1 - 
1 - OK
1 - CAMERA ON
1 - 2014-09-09T19:06:43.515 Python Client: initialized by teami@instrument disconnect - Disconnect called for client
